README.md

```diff
@@ -14,6 +14,7 @@
   - [Authenticating to Azure](./docs/deployment/deployment.md#log-into-azure-using-the-azure-cli)
   - [Deploying to Azure](./docs/deployment/deployment.md#deploy-and-configure-azure-resources)
   - [Troubleshooting Deployment & Startup Issues](./docs/deployment/worbook_usage.md)
+  - [Considerations for Production Adoption](./docs/deployment/deployment.md#additional-considerations-for-a-production-adoption)
```

Comment:
I feel this should point directly to the document, not a paragraph in another page with a link to the document.